memphis/memphis.py

```diff
@@ -599,15 +612,21 @@ async def consumer(
             create_consumer_req_bytes = json.dumps(create_consumer_req, indent=2).encode(
                 "utf-8"
             )
-            err_msg = await self.broker_manager.request(
+            creation_res = await self.broker_manager.request(
                 "$memphis_consumer_creations", create_consumer_req_bytes, timeout=5
             )
-            err_msg = err_msg.data.decode("utf-8")
+            creation_res = creation_res.data.decode("utf-8")
+            creation_res = json.loads(creation_res)
 
-            if err_msg != "":
-                raise MemphisError(err_msg)
 
+            if creation_res["error"] != "":
+                raise MemphisError(creation_res["error"])
             internal_station_name = get_internal_name(station_name)
+
+            if not creation_res["partitions_update"]["partitions_list"] is None:
```

Comment:
please move the "not" before the "None":

`if creation_res["partitions_update"]["partitions_list"] is not None:`